examples/sktime/flavor.py

```diff
@@ -468,7 +469,7 @@ class _SktimeModelWrapper:
     def __init__(self, sktime_model):
         self.sktime_model = sktime_model
 
-    def predict(self, dataframe) -> pd.DataFrame:
+    def predict(self, dataframe, params: Optional[Dict[str, Any]] = None) -> pd.DataFrame:
```

Comment:
For sklearn, `params` is just for compatibility?

Reply:
We might want to introduce a base class for wrappers.

Reply:
Yea it's for compatibility, because PyFuncModel's predict method must have `params` in signature. What does this `We might want to introduce an abstract class for wrappers to clarify the contract.` mean?

Reply:
For example, if we have a class like this, that would clarify what each wrapper needs to implement:

```python
class BaseWarapper(ABC):
    def predict(self, dataframe, params):
        ...
```

Reply:
Ah I see, this makes sense. I'll create one base wrapper and make all flavors inherit it.